public/css/quizzard.css

```diff
@@ -3,3 +3,32 @@ body {
   padding-bottom: 20px;
   background-color: #eee;
 }
+
+#snackbar {
+    visibility: hidden;
+    min-width: 500px;
+    margin-left: -250px;
+    color: #000000;
+    text-align: center;
+    border-radius: 20px;
+    padding: 8px;
+    position: fixed;
+    z-index: 1100;
+    left: 50%;
+    top: 7px;
+}
+
+#snackbar.show {
+    visibility: visible;
+    animation: fadein 0.5s, fadeout 0.5s 2.5s;
```

Comment:
Does this animation show in all major browsers?

Reply:
Good thinking, let's get this tested in IE Firefox Chrome and Safari

Reply:
FireFox and Chrome are good.

Reply:
Could we get @mehtadarshan1 (or @larryyueli ) to fork my repo at some point and tests if it works on safari? And @ahmedqarmout2 or @esteeroramy you guys work on windows so can you test IE?

How to test:
1. Fork my repo. ( https://github.com/Mohammed-JS-Khan/quizzard )
2. Pull on to your laptop, make an admin account and log in using Safari or IE.
3. Go to student and create a valid student.
4. Confirm if a green bubble slides down from the top, hangs for a second, then slides back up.

I'd do this myself but I don't have Windows or OSX and I can't find IE or Safari on the Arch User Repo.

Reply:
Worked on IE and Safari it looks good